draft-irtf-cfrg-frost.md

```diff
@@ -865,6 +868,18 @@ signature under the group public key. However, since the aggregate signature is
 and only if all signature shares are valid, this order of operations is more expensive
 if the signature is valid.
 
+## Identifiable Abort {#abort}
+
+FROST does not provide robustness. As such, a misbehaving participant can cause a denial-of-service
+on the signing protocol if it is allowed to contribute to the signing protocol.
+Preventing this type of attack requires the Coordinator to identify the misbehaving
```

Comment:
```suggestion
FROST allows for identifying misbehaving participants that caused the protocol to abort; preventing this attack in future runs of the protocol requires the Coordinator to honestly report the misbehaving
```

Reply:
I don't agree with this suggestion, since (1) it assumes that the underlying channel has been authenticated as is required for public key shares to be associated with the desired entities and (2) we haven't said what "honestly reporting" even means.